crates/fluvio-sc/src/controllers/mirroring/controller.rs

```diff
@@ -174,12 +96,81 @@ impl<C: MetadataItem> RemoteMirrorController<C> {
 
             debug!(
                 "sleeping {} seconds before next iteration",
-                MIRRORING_CONTROLLER_INTERVAL
+                MIRRORING_CONTROLLER_INTERVAL_MS
             );
-            sleep(Duration::from_secs(MIRRORING_CONTROLLER_INTERVAL)).await;
+            sleep(Duration::from_millis(MIRRORING_CONTROLLER_INTERVAL_MS)).await;
```

Comment:
is there reason why this is so small?

Reply:
This is the interval of remote SC connecting to home SC. It's a live forever connection that home SC response. So, it's very interesting this have a minor interval to connect as soon as possible as the home SC is handling the real interval, which now is 5 seconds.

Also, there is another interval of 10 seconds in the outside loop when there is an error (connection close, for example) or there is no remote registered.

I also notice a lot of `err: Socket io Broken pipe (os error 32)` on the home SC for the first requests, and for some reason this returns None instead of an error for the stream. Maybe a bug on our stream?

So with a small interval here, I noticed the first connection went quickly.